src/amo/reducers/ratings.js

```diff
@@ -0,0 +1,25 @@
+import { SET_USER_RATING } from 'amo/constants';
+
+export const initialState = {
+  userRatings: {},
+};
+
+export default function ratingsReducer(state = initialState, action) {
```

Comment:
I think `Reducer` is normally left out so this should instead be called `ratings`.